larray/core.py

```
@@ -10381,6 +10396,8 @@ def full(axes, fill_value, title='', dtype=None, order='C'):
          BE | 0 | 1
          FO | 0 | 1
     """
+    if isinstance(fill_value, Axis):
+        raise TypeError("`fill_value` must be a numeric value")
```

wouldn't it be better to raise the same error than other creation functions?